packages/grid/_modules_/grid/hooks/features/columns/gridColumnsUtils.ts

```diff
@@ -100,6 +102,23 @@ export const getGridColDef = (
   if (!type) {
     return columnTypes[DEFAULT_GRID_COL_TYPE_KEY];
   }
+
+  if (process.env.NODE_ENV !== 'production') {
+    if (!columnTypeWarnedOnce && !columnTypes[type]) {
+      console.warn(
+        [
+          `MUI: The column type "${type}" you are using is not supported.`,
+          `Column type "${DEFAULT_GRID_COL_TYPE_KEY}" is being used instead.`,
```

Comment:
It's printing "\_\_default\_\_" but we never mention in the docs what is "\_\_default\_\_". Although, we do mention that "string" is the default column type: https://deploy-preview-3757--material-ui-x.netlify.app/components/data-grid/columns/#column-types

```suggestion
          `Column type "string" is being used instead.`,
```